download_cmd.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/facebookgo/stackerr"
+	"github.com/spf13/cobra"
+)
+
+type downloadCmd struct{}
```

Comment:
thanks for working on this.

`parse download` is a bit ambiguous imho.
these are the two cases one might want:
1) set up a totally new folder with code for an app (much like clone)
2) rewrite code in existing directory with code from the app (much like pull & reset)

for 1) you should integrate this functionality into new workflow, since it sets up keys for the app as well
for 2) you could just integrate into the releases command
say parse releases -v{version} --download

either way, I think you should also work on being able to pull individual releases & not just the latest release.
(since parse supports last 10 releases, you should be able to pull those)

I think `parse